src/services/telementryService.js

```
@@ -1,217 +1,257 @@
+  contentSessionId = localStorage.getItem("contentSessionId");
 } else {
-    contentSessionId = localStorage.getItem('virtualStorySessionID') || uniqueId();
-    localStorage.setItem('allAppContentSessionId', contentSessionId);
+  contentSessionId =
+    localStorage.getItem("virtualStorySessionID") || uniqueId();
+  localStorage.setItem("allAppContentSessionId", contentSessionId);
 }
 
 let getUrl = window.location.href;
-url = getUrl && getUrl.includes('#') && getUrl.split('#')[1].split('/')[1];
+url = getUrl && getUrl.includes("#") && getUrl.split("#")[1].split("/")[1];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)